resources/lambda_functions.go

```diff
@@ -916,6 +916,19 @@ func LambdaFunctions() *schema.Table {
 					},
 				},
 			},
+			{
+				Name:        "aws_lambda_runtimes",
```

Comment:
why not just embed as a text array? also what is the relational FK here? can each function have only 1 runtime? if so just save it has a text field as runtime? Basically you will call this resolve each time per function and it will all conflict each time.